scope/init.go

```diff
@@ -123,3 +126,23 @@ func Init(my *Extent, create bool) (*Extent, error) {
 
 	return sv, nil
 }
+
+func setDefaultVersion(my, sv *Extent, version string) error {
+	var (
+		initVersion Version
+		err         error
+	)
+	if version == "" {
+		// default version is 0.0.0
+		initVersion = Version{}
```

Comment:
Where are you setting the "default version" to 0.0.0?

Reply:
> Where are you setting the "default version" to 0.0.0?

calling the Version struct literal without any keys returns the semantic version 0.0.0.  The existing WriteVersion method below expects a Version type as its third argument.

Reply:
yea, it is from the definition of original Version struct of blang.gitsemver - see this: https://github.com/blang/semver/blob/master/semver.go#L24-L30

Reply:
btw, we are still using the old version of gitsemver. any plan to update to the latest version 3.6.1?

Reply:
No plans yet, as things are right now, due to the lack of unit tests/coverage, I only want to make minimal changes to the code.  I haven't looked at the release notes for semver 3.6.1 is there any functionality there that you think we should be consuming?

Reply:
probably no functionality we needed unless we are doing the incremental versioning using the latest API from 3.6.0.   See the attached screenshot I took from their github release
![image](https://user-images.githubusercontent.com/25697718/75060621-68512080-549c-11ea-8dad-643cca4d005b.png)

Reply:
I will add a story in our backlog to update with latest semver. Thanks.